### PR TITLE
Feature/item slider

### PR DESCRIPTION
**Pull request by voonya**

### CODE REVIEW

File: packages/frontend/components/item/image-modal/component.tsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+  image,
+  isOpen,
+  setModalVisible,
+}: ImageModalProps) => (
+  <Modal visible={isOpen}>
+    <div css={styles.modalImageWrapper}>
+      <img src={image} alt="item" css={styles.modalImage} />
+      <div css={styles.modalClose}>
+        <IconButton
+          ariaLabel="closeModal"
```
Comment (Nightlngale) [inline packages/frontend/components/item/image-modal/component.tsx:21]: use loc. key

File: packages/frontend/components/item/image-slider/component.tsx [hunk 2]
```diff
@@ -8,38 +8,58 @@ interface ItemImageSliderProps {
...
     setFocusedImage(imageLink);
   };
 
+  useEffect(() => {
+    function handleWheelScroll(e) {
+      e.preventDefault();
+      imagesWrapper.current.scrollLeft += e.deltaY;
+      imagesWrapper.current.scrollTop += e.deltaY;
+    }
+    imagesWrapper.current.addEventListener('wheel', handleWheelScroll);
```
Comment (Nightlngale) [inline packages/frontend/components/item/image-slider/component.tsx:24]: you need to remove an event listener on willUnMount

Comment (Nightlngale) [reply packages/frontend/components/item/image-slider/component.tsx:24]: and what does it do?

File: packages/frontend/components/item/image-slider/component.tsx [hunk 3]
```diff
@@ -8,38 +8,58 @@ interface ItemImageSliderProps {
...
+        onClick={() => setIsModalOpen(true)}
+        aria-hidden="true"
+      >
+        <img src={imageLinks[focusedImage]} alt="item" css={styles.image} />
+        <div
+          css={styles.seeImageCaption}
+          onClick={() => setIsModalOpen(true)}
+          aria-hidden="true"
+        >
+          Open full image
```
Comment (Nightlngale) [inline packages/frontend/components/item/image-slider/component.tsx:55]: localisation

File: packages/frontend/components/item/image-slider/component.tsx [hunk 4]
```diff
@@ -8,38 +8,58 @@ interface ItemImageSliderProps {
...
-      <div css={styles.focusedImage}>
-        <Image
-          src={imageLinks[focusedImage]}
-          alt="item image"
-          layout="fill"
-          objectFit="cover"
-        />
+      <div
+        css={styles.focusedImageWrapper}
+        onClick={() => setIsModalOpen(true)}
```
Comment (Nightlngale) [inline packages/frontend/components/item/image-slider/component.tsx:46]: It's bad practice to have clickable div. But you are working in that way, no need to provide role, tabIndex and onKeyDown handler

**Review by Nightlngale — CHANGES_REQUESTED** [review]

File: packages/frontend/components/item/image-slider/component.tsx (hunk 2 above)
Comment (voonya) [reply packages/frontend/components/item/image-slider/component.tsx:24]: add posibility to scroll horizontal scroll bu wheel

Comment (Nightlngale) [reply packages/frontend/components/item/image-slider/component.tsx:24]: doesn't it work with shift/ctrl pressed?

File: packages/frontend/components/item/image-slider/styles.ts [hunk 5]
```diff
@@ -25,51 +25,93 @@ export const sliderWrapper = ({ spaces, breakpoints }: Theme) => css`
 export const imagesWrapper = ({ spaces, breakpoints }: Theme) => css`
   display: flex;
   flex-direction: column;
-  justify-content: flex-start;
-  margin-right: ${spaces.lg};
-  width: 100px;
-
+  justify-content: fle;
```
Comment (YevheniiBieliaiev) [inline packages/frontend/components/item/image-slider/styles.ts:28]: need to fix value in justify-content

File: packages/frontend/components/item/image-modal/component.tsx [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+          icon={IconName.XMARK}
+          color={IconColor.ORANGE}
+          backgroundColor={'lightgray'}
+          size="md"
```
Comment (YevheniiBieliaiev) [inline packages/frontend/components/item/image-modal/component.tsx:25]: maybe we need to add enum for this type of colors?

Comment (voonya) [reply packages/frontend/components/item/image-modal/component.tsx:25]: we already have type 
image

**Review by YevheniiBieliaiev — APPROVED** [review]

**Review by Nightlngale — APPROVED** [review]

**Pull request merged**